docs/PARAMETERS.md

```diff
@@ -12,23 +12,27 @@ Brief description of the parameters for configuring mLRS. The parameters fall in
 
 All parameters are configured via the Tx module, by using the CLI or the mLRS configuration Lua script (or the OLED if available). If no receiver is connected, then the Rx parameters cannot be configured.
 
-Depending on the specific mLRS hardware, it may happen that some parameters are not available, as well as that for a parameter some options are not available.
+Depending on the specific mLRS hardware, it may happen that some parameters are not available or that for a parameter some options are not available.
 
 ## Common Parameters ##
 
 #### Bind Phrase ####
 String of 6 characters. 
 The characters can be 'a'-'z', '0'-'9', '_', '#', '-', '.'. 
 
-For 2.4 GHz systems the last (6th) character of the bind phrase determines excluded frequencies. The exclude setting can be "\e-" (no frequencies excluded), "\e1" (wlan band #1 excluded), "\e6" (wlan band #6 excluded), "\e11" (wlan band #11 excluded), or "\e13" (wlan band #13 excluded), and it is determined from the last character as follows: 'a' = "\e-", 'b' = "\e1", 'c' = "\e6", 'd' = "\e11", 'e' = "\e13", and with 'f' the cycle continues, and so forth.
+For 2.4 GHz systems the last (6th) character of the bind phrase determines excluded frequencies. The exclude setting can be "\e-", "\e1", "\e6", "\e11", or "\e13". For details see [FHSS_SHAPING](FHSS_SHAPING.md).
 
 #### Mode ####
-Operation mode. 
-Can be "50 Hz", "31 Hz", "19 Hz", "FLRC".
+Operation mode. May not be selectable.
+Can be "50 Hz", "31 Hz", "19 Hz", "FLRC", "FSK".
 
 #### RF Band ####
 Frequency band. May not be selectable.
 
+#### RF Ortho ####
+Allows to constrain the used frequencies in the FHSS sequence, see [FHSS_SHAPING](FHSS_SHAPING.md). May not be available.
```

Comment:
As opposed to 'May not be available' - could be 'For 2.4 GHz and 915 MHz RF bands allows one to constrain...' - similar to sentence above for bind phrase.